Biklas_API_V2/Controllers/UsuariosController.cs

```diff
@@ -11,19 +11,51 @@ namespace Biklas_API_V2.Controllers
     public class UsuariosController : BKBaseApiController
     {
         // GET api/<controller>
-        public IHttpActionResult Get()
+        /// <summary>
+        /// Devuelve al cliente la lista de usuarios almacenados en la BD.
+        /// Se debe especificar el id del usuario que realiza la búsqueda,
+        /// dicho usuario se excluye de los resultados. También se puede 
+        /// especificar un texto de búsqueda el cual se usará como filtro
+        /// para obtener los resultados.
+        /// </summary>
+        /// <param name="idUsuario">El id del usuario que realiza la búsqueda</param>
+        /// <param name="textoBusqueda">El texto de búsqueda de usuarios</param>
+        /// <returns></returns>
+        public IHttpActionResult Get(int idUsuario, string textoBusqueda = null)
         {
-            List<Usuarios> usuarios = db.Usuarios.ToList();
+            // La búsqueda se estandariza a minúsculas
+            textoBusqueda = textoBusqueda?.ToLower();
+
+            // El usuario de búsqueda se excluye de los resultados...
+            IEnumerable<Usuarios> usuarios = db.Usuarios.Where(u => u.IdUsuario != idUsuario);
+
+            if (textoBusqueda != null && !string.IsNullOrWhiteSpace(textoBusqueda))
```

Comment:
No es necesario comparar con null, ya que se utiliza la función 'IsNullOrWhiteSpace'
```suggestion
            if (!string.IsNullOrWhiteSpace(textoBusqueda))
```